src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterlab/overrides.json

```
@@ -61,6 +86,21 @@
                         }
                     }
                 ]
+            },
+            {
+                "id": "jp-mainmenu-help",
+                "rank": 1001,
+                "items": [
+                    {
+                        "command": "help:open",
+                        "rank": 1001,
+                        "args": {
+                            "url": "https://www.nebari.dev/docs/welcome",
```

```suggestion
                            "url": "https://www.nebari.dev/docs/welcome/",
```

This URL redirects to the one with slash, so let's save users on slower connections some time.